internal/component/beyla/ebpf/beyla_linux.go

```diff
@@ -551,14 +560,11 @@ func (args *Arguments) Validate() error {
 		}
 	}
 
-	// Check that at least one feature type is enabled
-	if !hasNetworkFeature && !hasAppFeature {
-		return fmt.Errorf("metrics.features must include at least one of: network, application, application_span, application_service_graph, or application_process")
+	// Check that at least one feature type is enabled or tracing is enabled
+	if !hasNetworkFeature && !hasAppFeature && !isTracingEnabled && !hasOutputConfig {
+		return fmt.Errorf("either metrics.features must include at least one of: network, application, application_span, application_service_graph, application_process, or tracing must be enabled via trace_printer or output section")
```

Comment:
Can we put the list in brackets `[]` or something to make this easier to read? As is it's hard to separate the metrics features list from the tracing message.